lib/utils/form-builder/groupedFormHelpers.ts

```diff
@@ -0,0 +1,57 @@
+import { Group, GroupsType } from "@lib/formContext";
+import { FormElement, FormProperties } from "@lib/types";
+
+export interface IFormElementHash {
+  [key: string]: FormElement;
+}
+
+// Returns the elements of a group in the order they are in the form as defined by group.
+export const sortGroup = ({ form, group }: { form: FormProperties; group: Group }) => {
+  const elementCollection: IFormElementHash = getFormElementCollection(form);
+  const sortedElements: FormElement[] = [];
+
+  if (!group.elements) return sortedElements;
+
+  const groupElements = group.elements.map((elId) => {
+    return elementCollection[elId];
+  });
+
+  sortedElements.push(...groupElements);
+
+  return sortedElements;
+};
+
+//----
+//TODO: Move this into the template store, and use the template store to get the the collection of elements.
```

Comment:
Can remove this comment --- we'll keep as a util vs moving to the template store